doc/hybrid_planning/hybrid_planning_tutorial.rst

```diff
@@ -0,0 +1,252 @@
+                plugin="moveit_hybrid_planning::GlobalPlannerComponent",
+                name="global_planner",
+                parameters=[
+                    global_planner_param,
+                    robot_description,
+                    robot_description_semantic,
+                    kinematics_yaml,
+                    ompl_planning_pipeline_config,
+                ],
+            ),
+            ComposableNode(
+                package="moveit_hybrid_planning",
+                plugin="moveit_hybrid_planning::LocalPlannerComponent",
+                name="local_planner",
+                parameters=[
+                    local_planner_param,
+                    robot_description,
+                    robot_description_semantic,
+                    kinematics_yaml,
+                ],
+            ),
+            ComposableNode(
+                package="moveit_hybrid_planning",
+                plugin="moveit_hybrid_planning::HybridPlanningManager",
+                name="hybrid_planning_manager",
+                parameters=[hybrid_planning_manager_param],
+            ),
+        ],
+        output="screen",
+    )
+
+An example launch file can be found in the :codedir:`tutorial's files <hybrid_planning/launch/hybrid_planning_tutorial.launch.py>`. If you copy&paste it don't forget to remove the :codedir:`Test node <hybrid_planning/launch/hybrid_planning_tutorial.launch.py#L180>` that implements the demo from above!
+
+Customizing the Hybrid Planning Architecture
+--------------------------------------------
+As the rest of Moveit 2, the *Hybrid Planning Architecture* is designed to be highly customizable while also offering the possibility to easily re-use existing solutions. Each of the architecture's components is a ROS 2 node and can be completely replaced by your own custom ROS 2 node as long as it offers the API required by the other nodes. Each component's runtime behavior is defined by plugins. This section focuses on the customization of the *Hybrid Planning Architecture* by implementing your own plugins.
+
+Global and Local Motion Planning
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To get a global motion planning solution, the *Global Planner Component* needs to be activated via the *Global Planning Action Server*. When it receives a *MotionPlanRequest* the component computes a motion plan with the *Global Planner Plugin* and publishes the solution to the other components.
+The dataflow within the component can be seen in the picture below:
+
+.. image:: images/global_planner_dataflow.png
+   :width: 500pt
+
+The *Global Planner Plugin* can be used to implement and customize the global planning algorithm. To implement you own planner you simply need to inherit from the :moveit2_codedir:`GlobalPlannerInterface <moveit_ros/hybrid_planning/global_planner/global_planner_component/include/moveit/global_planner/global_planner_interface.h>`: ::
+
+   class MySmartPlanner : public GlobalPlannerInterface
```

Comment:
Consider renaming `GlobalPlannerInterface` to `GlobalPlannerPlugin` to make it clear this is the plugin interface for a global planner.